packages/orbit-components/src/Radio/README.md

```diff
@@ -16,28 +16,29 @@ After adding import into your project you can use it simply like:
+| :------------- | :------------------------- | :------ | :-------------------------------------------------------------------------------------------------------- |
+| checked        | `boolean`                  | `false` | If `true`, the Radio will be checked.                                                                     |
+| defaultChecked | `boolean`                  |         | If `true`, the Radio will be checked by default. Only to be used in uncontrolled.                         |
+| dataTest       | `string`                   |         | Optional prop for testing purposes.                                                                       |
+| id             | `string`                   |         | Set `id` for `Radio` input                                                                                |
+| disabled       | `boolean`                  | `false` | If `true`, the Radio will be set up as disabled.                                                          |
+| hasError       | `boolean`                  | `false` | If `true`, the border of the Radio will turn red. [See Functional specs](#functional-specs)               |
+| info           | `React.Node`               |         | The additional info about the Radio.                                                                      |
+| label          | `string`                   |         | The label of the Radio.                                                                                   |
+| name           | `string`                   |         | The name for the Radio.                                                                                   |
+| onChange       | `event => void \| Promise` |         | Function for handling onChange event.                                                                     |
+| ref            | `func`                     |         | Prop for forwarded ref of the Radio. [See Functional specs](#functional-specs)                            |
+| tabIndex       | `string \| number`         |         | Specifies the tab order of an element                                                                     |
+| tooltip        | `Element<Tooltip>`         |         | Optional property when you need to attach Tooltip to the Radio. [See Functional specs](#functional-specs) |
+| value          | `string`                   |         | The value of the Radio.                                                                                   |
+| readOnly       | `boolean`                  |         | If `true`, the Radio will be set up as readOnly.                                                          |
 
 ## Functional specs
 
-- The`hasError` prop will be visible only when the Radio has `checked` or `disabled` prop set on false.
+- The`hasError` prop will be visible only when the Radio is not checked or disabled.
```

Comment:
nitpick:
```
checked nor disabled.
```
as it's written it `Checkbox/README.md`

Reply:
Thanks